func_sig_registry/registry/serializers.py

```diff
@@ -82,31 +85,36 @@ class SolidityImportSerializer(serializers.Serializer):
     num_processed = serializers.IntegerField(read_only=True)
     num_imported = serializers.IntegerField(read_only=True)
     num_duplicates = serializers.IntegerField(read_only=True)
+    num_ignored = serializers.IntegerField(read_only=True)
 
     def create(self, validated_data):
-        import_results = []
+        stats_function = empty_import_stats()
```

Comment:
*nitpick*

I think it would be better to place these in `else` blocks with their corresponding `if` blocks below rather than overwriting the variable.

Functionally these are the same, however in my experience I have found that *treating* python as immutable whenever possible reduces a number of ways that bugs can creep into code, thus I tend to avoid destructive assignments like this.

Reply:
I see. Actually I realized that implementation is wrong. The previous one allowed for requests to have both `source_file ` and `source_code`, however, the pending change overwrites the other if both exist.

Reply:
I would recommend writing a test first that would have exposed this error, prior to fixing it.